reactDashboard/src/components/Facts/Facts.jsx

```diff
@@ -6,52 +6,59 @@ const Facts = () => {
     <div className="facts-container">
       <div className="facts-row">
         <div className="facts-component">
-          <img className="facts-icon" src="./assets/wear_face_coverings_no_bg.png" alt="" />
+          <img
+            className="facts-icon"
+            src="./assets/wear_face_coverings_no_bg.png"
+            alt="two faces wearing masks"
```

Comment:
I'm not sure alt text is advisable here - the icons are decorative and do not supply any more information than the text in each box. Having a screenreader read them out would make it **less** accessible I think...